hack/e2e.sh

```diff
@@ -37,10 +41,6 @@ function debug_navigator_start() {
 }
 
 function helm_install() {
-    if [ "${CHART_VALUES}" == "" ]; then
-        echo "CHART_VALUES must be set";
-        exit 1
-    fi
```

Comment:
Is this no longer required?

Reply:
Nope. The `: ${CHART_VALUES:?}` causes the script to exit with almost that exact message if the variable isn't set.